### ISSUE TITLE
Permissions Error with tom_setup

### ISSUE THREAD

**Issue by rachel3834** [TOMToolkit/tom_base#77]
An external user has been working through our Getting Started guide on Mac OS, but encountered the error below, which I re-create if I follow the instructions exactly.

(tom_env) Rachels-MacBook-Pro-2:mytom rstreet$ ./manage.py tom_setup
Welcome to the tom_setup helper script. This will help you get started with a new TOM.
DO NOT RUN THIS SCRIPT ON AN EXISTING TOM. It will override any custom settings you may already have.
Do you wish to continue? [Y/n] Y
Checking Python version... OK
Creating project directories... Traceback (most recent call last):
  File "./manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/django/core/management/base.py", line 316, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/django/core/management/base.py", line 353, in execute
    output = self.handle(*args, **options)
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/tom_setup/management/commands/tom_setup.py", line 126, in handle
    self.create_project_dirs()
  File "/Users/rstreet/software/tom_env/lib/python3.7/site-packages/tom_setup/management/commands/tom_setup.py", line 69, in create_project_dirs
    os.mknod(os.path.join(BASE_DIR, 'static/.keep'))
PermissionError: [Errno 1] Operation not permitted

**Comment by Fingel** [comment]
Fixed. This was due to os.mknod requiring superuser privileges on OSX. Would be nice if an OSX user could confirm the fix. 

https://stackoverflow.com/questions/32115715/os-mknod-fails-on-macos

**Comment by rachel3834** [comment]
I just tested this with a from-scratch install and I think the fixed has worked; instead I ran into what I think is a different problem.  When I ran the migration immediately after running the tom_setup, I got this:

(tom_env) Rachels-MacBook-Pro-2:mytom rstreet$ ./manage.py migrate
SystemCheckError: System check identified some issues:

ERRORS:
tom_dataproducts.ReducedDatum.data_type: (fields.E005) 'choices' must be an iterable containing (actual value, human readable name) tuples.

Possibly related to opting not to set a user password during the setup process?  Its the only thing I did differently.
